.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ references:
         GITHUB_TEAM_NAME_SLUG: laa-apply-for-legal-aid
     - image: postgres:10.5
     - image: redis:5.0
-    - image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-apply-for-legal-aid/clamav:1.0.0
+    - image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/laa-apply-for-legal-aid/clamav:1.0.2
```

Comment:
Using version `1.0.2` of ClamAV (https://github.com/ministryofjustice/clamav/releases)
The only difference with `1.0.0` is that it does not run as root.
(`1.0.1` was a failed attempt)